src/styles/_catalog.scss

```diff
@@ -1,9 +1,21 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  align-content: center;
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (max-width: 487px) {
+    grid-template-columns: 200px;
+  }
+  @media (min-width: 488px) and (max-width: 767px) {
```

Comment:
You can keep only min-width options. Don't need to combine it with max-width.

Reply:
maybe yes but
if I'll do the way you suggesting me
i'll have a strong need to watch out that next resolution is going exactly next in cascading style sheet file

do you really think I actually have to do all as short as it possible with risk of broking styles scenario?

Reply:
max-width conditional statements had been removed because of useless